O/oxigraph_server/build_tarballs.jl

```diff
@@ -16,7 +16,10 @@ cd $WORKSPACE/srcdir/oxigraph/server
 
 cargo build --release --no-default-features --features=sled
 
-install_license $WORKSPACE/srcdir/oxigraph/LICENSE-MIT
+cargo install cargo-license
+cargo-license --avoid-build-deps --avoid-dev-deps --json > CARGO_LICENSES.json
```

Comment:
You can't run programs built for the target.  Maybe this should built in a separate package and be a `HostBuildDependency`?